gapis/api/vulkan/frame_loop.go

```diff
@@ -478,9 +478,13 @@ func (f *frameLoop) Transform(ctx context.Context, cmdId api.CmdID, cmd api.Cmd,
 
 				// Add unconditional jump instruction to bring us back to the start of the loop if we made it past the conditional break.
 				stateBuilder.write(stateBuilder.cb.Custom(func(ctx context.Context, s *api.GlobalState, b *builder.Builder) error {
-					b.Push(value.S32(1))
```

Comment:
You plan on reverting the loop structure to before I fixed the last iteration rewind problem?